package/rmt-server-rpmlintrc

```diff
@@ -1,12 +1,9 @@
 addFilter("non-standard-uid")
 addFilter("script-without-shebang")
-addFilter("hidden-file-or-dir /usr/share/rmt/*")
+addFilter("hidden-file-or-dir /usr/share/rmt/.bundle")
 addFilter("hidden-file-or-dir /usr/lib64/rmt/vendor/bundle/*")
 addFilter("unexpanded-macro /usr/lib64/rmt/vendor/bundle/*")
 addFilter("zero-length /usr/lib64/rmt/vendor/bundle/*")
-addFilter("wrong-script-end-of-line-encoding /usr/lib64/rmt/vendor/bundle/*")
-addFilter("hidden-file-or-dir /usr/lib64/rmt/.bundle")
 addFilter("file-not-in-%lang /usr/lib64/rmt/vendor/bundle/*")
 addFilter("no-soname /usr/lib64/rmt/vendor/bundle/*")
 addFilter("non-executable-script /usr/lib64/rmt/vendor/bundle/*")
-addFilter("E: env-script-interpreter .* /usr/lib64/rmt/vendor/bundle/*")
```

Comment:
Is this warning really gone?

Reply:
yes, this rpmlin is already pushed to: https://build.opensuse.org/package/show/systemsmanagement:SCC:RMT/rmt-server

Reply:
Cool :+1: